dk.sdu.se4.keyboardinput/src/main/java/dk/sdu/se4/keyboardinput/KeyboardInputImpl.java

```diff
@@ -0,0 +1,74 @@
+package dk.sdu.se4.keyboardinput;
+
+import dk.sdu.se4.commongameinput.GameInput;
+import dk.sdu.se4.commongameinput.GameInputKeys;
+import java.util.HashMap;
+import java.util.Map;
+import com.badlogic.gdx.Input.Keys;
+import com.badlogic.gdx.InputAdapter;
+
+public class KeyboardInputImpl extends InputAdapter implements GameInput {
+    private final Map<GameInputKeys, Boolean> keys = new HashMap();
+    
+    public boolean isPressed(GameInputKeys key) {
+        if (keys.get(key)) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    @Override
+    public boolean keyDown(int keycode) {
+        //System.out.println("DOWN " + Keys.toString(keycode));
+        switch (keycode) {
```

Comment:
Dont use switch statements

Reply:
Why not?

Reply:
code smell

Reply:
en måde at løse det på er at lave det om til en liste af keys og validate dem